interfaces/Portalicious/src/app/pages/login/login.component.html

```diff
@@ -49,6 +49,7 @@ <h1 i18n>Log in</h1>
               autocomplete="current-password"
               [toggleMask]="true"
               [feedback]="false"
+              styleClass="block w-full"
```

Comment:
Rather than duplicating this (I wonder if it is necessary also on the change-password page, for example?) I think it would be easier to leave the rules that were removed in `form-field-wrapper` component:

`<label class="[&_.p-password]:block [&_input]:w-full">`

Reply:
My thinking was that this place(the login-page) is the only exception to the rule. So therefore I don't want to over-complicate the reusable component and keep that generic/simple.

Reply:
> My thinking was that this place(the login-page) is the only exception to the rule.

That assumption is incorrect. The change password page is broken on the preview deployment.